src/main/java/Calculator.java

```diff
@@ -0,0 +1,72 @@
+import java.util.Scanner;
+
+public class Calculator {
+    public void Calculator(){};
+    public int operation(String[] values)throws  Exception{
+        this.validation(values);
+
+        return filtNumberAndOperation(values);
+    }
+
+    public int filtNumberAndOperation(String[] values){
+        int result=0;
+        for(int i=0;i<values.length-2;i+=2){
+            result = classifyIndex(i,result,values);
+        }
+        return result;
+    }
+
+    public int classifyIndex(int i, int result, String[] values){
+        int firstNumber = Integer.parseInt(values[i]);
+        int secondNumber = Integer.parseInt(values[i+2]);
+        String op = values[i+1];
+        if(i==0){
+            result = operating(firstNumber,secondNumber,op);
+        }else{
+            result = operating(result,secondNumber,op);
+        }
+        return result;
```

Comment:
```suggestion
        if(i==0){
            return operating(firstNumber,secondNumber,op);
        }
        return operating(result,secondNumber,op);
```
이 메소드를 호출하는 로직을 조금 수정하면 이렇게 else문 없이 할 수 있겠죠?

Reply:
아 else문 제거를 신경을 못썼습니다..